tests/kernel/kdump_over_nfs.pm

```diff
@@ -0,0 +1,53 @@
+# SUSE's openQA tests
+#
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Configure and run kdump over NFS.
+# The test is configuring - on the nfs client side - the kdump with
+# KDUMP_SAVEDIR set to NFS share. Then the actual crash is triggered,
+# still on the NFS client side and once the reboot happens, the nfs share,
+# set as the target in the KDUMP_SAVEDIR, is mounted and the crash files
+# are checked
+#
+# Maintainer: QE Kernel <kernel-qa@suse.de>
+
+use base "opensusebasetest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use kdump_utils;
+use lockapi;
+use power_action_utils 'power_action';
+
+sub run {
+    my ($self) = @_;
+    my $role = get_required_var('ROLE');
+    my $kdump_export = get_required_var('KDUMP_SAVEDIR');
+    my $nfs_server = get_var('NFS_SERVER') // 'server-node00';
+
+    select_console('root-console');
+
+    if ($role eq 'nfs_client') {
+        assert_script_run("mkdir -p /var/crash");
+        assert_script_run("echo \"$nfs_server:/nfs/shared_nfs3 /var/crash nfs nfsvers=3,sync,nofail,x-systemd.automount 0 0\" >> /etc/fstab");
+        assert_script_run("mount -a");
```

Comment:
What about `nfsvers=4` ?

We have variable `KDUMP_SAVEDIR="nfs://server-node00/nfs/shared_nfs3"` but here we have path hardcoded.

I see both points as content for follow up ticket.

Reply:
Added this comment to the progress ticket, so setting this as resolved